[Content] The Avgi - Atomic Space Dragonflies, In Space (#9172)
endless-sky/endless-sky: data/avgi/avgi side missions.txt

			choice
				`	"Sure."`
				`	"No thank you."`
					defer
			`	"Delight. The youth these days have little appreciation for the art of Trefoil, you know." He taps one eye knowingly, an odd gesture. "Or at least my mother would say. That was sixty years ago, hmm. Please, sit, sit!" He points to the other side of his bench, with nothing more than a horizontal rail for a seat.`
			choice
				`	"That sure looks comfortable."`
				`	"I'll stand, I think."`
			`	"Amusement. The other humans had chairs made to suit your forms, they did. Folded them up and carried them everywhere."`
			`	"Did you meet them?" you ask.`
			`	"Yes, yes. Several times. Only the angry one was any good at Trefoil, sadly. Speaking of which -" he deftly rearranges the colored stones, arranging them into a neat pattern. Three triangles, one of each color, angled with their topmost points touching at the center of a six-sided board.`

			scene "scene/trefoil"

			`	"Respectful deference. Which color would you like to play as, question? There is no real difference between them, as far as the game plays."`
			choice
				`"Red"`
				`"Green"`
				`"Blue"`

> 			`	"Acknowledgment. Excellent. Explanation. The objective is to remove all of your pieces from the board by moving them to the opposite edge to your own triangle" he explains, demonstrating with a stone, then showing you how to skip it over connected chains of other pieces, moving to their endpoint. "You must balance your moves between freeing your own pieces and blocking the other colors from their own escape.`

Reviewer: ```suggestion
			`	"Acknowledgment. Excellent. Explanation. The objective is to remove all of your pieces from the board by moving them to the opposite edge to your own triangle," he explains, demonstrating with a stone, then showing you how to skip it over connected chains of other pieces, moving to their endpoint. "You must balance your moves between freeing your own pieces and blocking the other colors from their own escape."`
```

			`	You delicately take a seat on the rail. It's at least only a little worse than a park bench, and you're thankful for the lower gravity on Peripheria.`
			choice
				`	"But there's three sides and only two of us."`

> 			`	"Approving. An astute observation. The third color can be played by a third person, in some variations, but in our case we take turns moving for it. It - the Third - is an external actor, under our influence yet outside our absolute control, often used as a pawn to block the other player's moves when opportune. Now, would you like to make the first move, question? Players generally either focus on either evacuating their own pieces or moving to block their opponent's path to escape "`

Reviewer: ```suggestion
			`	"Approving. An astute observation. The third color can be played by a third person, in some variations, but in our case we take turns moving for it. It - the Third - is an external actor, under our influence yet outside our absolute control, often used as a pawn to block the other player's moves when opportune. Now, would you like to make the first move, question? Players generally either focus on either evacuating their own pieces or moving to block their opponent's path to escape."`
```

			choice
				`(Focus on getting your stones out. Move a stone towards your color's escape.)`
					goto escape1
				`(Focus on blocking Wisteria. Move a stone towards his triangle.)`
					goto blockwisteria1
				`(Focus on blocking the third color. Move a stone towards its triangle.)`
					goto blockthird1

			label escape1
			action
				"avgi: trefoil classical" ++
			`	"Approving. A good approach. That is the classical move to make at the start of the game, especially for a beginner."`
				goto next1

			label blockwisteria1
			action
				"avgi: trefoil aggressive" ++
			`	"Amusement. Ah, already moving to block me, I see. That is often an approach taken by more advanced players, but I must warn you, executing it successfully is difficult."`
				goto next1

			label blockthird1
			action
				"avgi: trefoil chaotic" ++
			`	"Interest. I do not often see players prioritize blocking the Third, especially as their first move. But perhaps you will surprise me."`
				goto next1

			label next1

> 			`	"Now, after each of our turns, we shall move one of the Third's pieces. You should focus on moving them to your advantage, whether that be by blocking my pieces or keeping them out of the way of your own."`

Reviewer: All of the labels that lead to this have a closing punctuation mark. Rather then adjusting all of those, how about this?
```suggestion
			`	Slightly adjusting his position on the bench, the Avgi continues. "Now, after each of our turns, we shall move one of the Third's pieces. You should focus on moving them to your advantage, whether that be by blocking my pieces or keeping them out of the way of your own."`
```

				goto next3

			label hardlife
			`	"Agreement. Yes, we do, and it grows harder the longer our war with the Aberrant goes on. We build our defenses up and they knock them down, we send armadas to retake Aviskir and they are routed or never return. It seems that all of our energies must go into this war, and little of our wealth is spent elsewhere. It is little wonder that some say we should seek to truly build a new life here, rather than waste our lives and efforts on reclaiming the old."`
				goto next3

			label safe
			`	"Amusement. Fair to say, as I suppose the perspective of an outsider must be more confused than enlightened, when confronted with a new species, a new civilization, for the first time. And I suppose you have seen little of our true civilization, the planets we left to the Aberrant."`

			label next3

> 			`	"Regardless, I wonder what you would think of us, had you arrived before the Aberrant threw us from our homes. Conflict. It is almost a shame to me that you have to see us in our current state, but then I think of the unity and selflessness that we have found in each other, and that fills me with pride."`

Reviewer: Same here.

		conversation
			`	You drop off Aliza near the same place you met her, in the main concourse. It's rather more full than last time, with a veritable crowd moving through the room, probably fresh off a transport ship. Aliza seems to spot someone she recognizes, waving, she drags you over to a dark blue Avgi in dark, steel gray clothing.`
			`	"This is my colleague, Yinmn," she says. And this is Captain <first> <last>.`
			`	"Greetings. Hey there. I work in the accounting section for Ergastirio. And you're of course Captain <last>, the alien everyone's been talking about."`
			choice
				`	"Huh, what do you do there?"`
					goto skip
				`	"Accounting? That's an odd job for a lab."`

			`	Yinmn hums with laughter. "Amusement, not that kind of accounting. I account for nuclear materials, not credits. We have to make sure every last drop of fissiles that could be used to make a bomb is kept secure. Not only for security reasons, but because it's extremely valuable out here in the Outer Limits. Not many of the heavier elements out here, sadly, and we've been having issues with shortages lately. It only takes a few kilos to make a bomb, but we do go through a lot of them..."`

> 

Reviewer 2: If I'm not mistaken, this line would go straight to line 1026, which would repeat the whole 'We have to make sure every last drop of fissiles...' dialog.

					goto believe
				`	"Very contradictory rumors."`

			`	She gives you an incredulous look. "Scorn. When are they not? I don't know about you humans, but ask three Avgi the same question and you'll get six different answers. We are a fractious lot, us Avgi, and that will be true until the stars burn out and the moons fall from the sky."`
				goto humans

			label believe
				`	"Amusement. Neither. Either one would be too easy, and that of course can't be allowed, not in this universe. I expect your Republic would spend years and years debating whether to even acknowledge our existence, let alone take any kind of action."`
			choice
				`	"You don't give the Republic enough credit. Parliament can be decisive when it needs to be."`
				`	"Parliament might be slow, but the Navy is full of good people who would help if they could."`
				`	"That's a surprisingly accurate estimation of the Republic."`
					goto accurate
				`	"Maybe, but there are others that would try to exploit you. That's just human nature."`
					goto exploit
				`	"Well, the Republic isn't the only government in human space these days. The Free Worlds might try to help you."`
					to display

> 						has "event: war begins"

Reviewer 3: It seems like most of this section's text does not display in the mission. (other than line 743) I am not well versed enough in mission construction to understand why though.

			
			label help
			`	"Dismissal. I've seen the <ship>, and the ship your friends arrived on. That's the kind of firepower we need to take on the Aberrant and win."`
			choice
				`	"If you say so."`
				`	"I hope you're right."`
			`	With that, you turn to lighter topics, as Claret finishes her drink. You learn about the new traditions the Twilight Guard is trying to promote to ensure unity and cohesion within its ranks, trying to bind together former members of dozens of independent militaries into a united whole. This apparently involves a lot of travel to promote inter-unit bonding, as well as frequent cycling between the front lines and the Outer Limits.`
			`	Eventually, Claret is ready to leave. "Anyway, I'll see you around. And, a word of advice. If you're ever on a Twilight Guard station, look out for officers or Guardsmen who are looking for a ride. We often have people trying to make an impromptu trip, and our logistics have been so overstretched lately they're willing to contract with outside captains, if they're trustworthy."`
			`	"I'll keep that in mind," you say.`
			decline
			action

> 				set "avgi: Twilight Guard Transport Jobs"

Reviewer 3: These don't fire. (the decline shows up as part of the conversation text.) Don't know why.

Reviewer: Because of capitalization.
```suggestion
			decline
			action
				set "Avgi: Twilight Guard Transport Jobs"
```

Reviewer 3: Thanks for catching that, though shouldn't the incorrect variable still get set here? Or is that not how it works?
(For context when I tested this the mission didn't decline until I removed one tab from the decline line, and the variable isn't present in my save no matter what I do.)

Reviewer: Hmm, good point.
Maybe this then?

```suggestion
			action
				set "Avgi: Twilight Guard Transport Jobs"
			decline
```

Reviewer 3: Yes, that works. Though I had to remove one tab from the decline to stop having it show up in the text. Dunno if that might cause other problems though.

	on offer
		conversation
			action
				set "Avgi: met wisteria"
			`As you explore the spaceport of <origin>, you come across a small grassy park, the greenspace a welcome contrast to the otherwise industrial facilities. As the centerpiece of the park, where you'd expect a fountain or a statue in a human park, is a curious hanging windchime, made of pieces of colorful glass and various metals. It's more than a little mesmerizing, and you stop to listen as it rustles in the gentle circulation of the station's air.`
			`	"Is this one of the humans I have heard so much about, question?" the translator tucked behind your ear suddenly speaks, responding to an Avgi voice. You turn and see an Avgi perched on a park bench behind you, his purple coloring faded with age. A board of colorful stones sits on the table in front of him, a game of some kind. He gives you an odd sort of salute with a forelimb, probably an Avgi greeting of some kind.`
			choice
				`	"Hello there."`
				`	"Hi, I'm Captain <first> <last>."`
					goto name
			`	"Courteous introduction. I am called Wisteria. Would you grace me with your own name?"`
			`	"I'm Captain <first> <last>," you say.`
			label name

> 			`	"Appreciated meeting. A pleasure to meet you, Captain <last>. Would you like to join old Wisteria at a game of Space Chess, question? I would be happy to teach you how to play, of course."`

Reviewer 2: Wait no
```suggestion
			`	"Appreciation. A pleasure to meet you, Captain <last>. Would you like to join old Wisteria at a game of Trefoil, question? I would be happy to teach you how to play, of course."`
```

				goto next

			label fun
			`	"Nah, we run all sorts of fun stuff that we can't test on Ergastirio, appreciation. The really big particle beams, for example, and reactor pumped lasers that would be annoying to shield on the station."`
			`	She gestures out the window, towards some stony mountains. "You can find a valley out that way with all sorts of tracks carved in the ice. I wrote my name in a patch of stone, actually.`
				goto next

			label next
			`	At Aliza's direction, you set the <ship> down on an icy pad by one of the small buildings. A few Avgi come out to greet you, wearing silver-colored spacesuits with comically large bubble helmets.`
			choice
				`	"What now?"`
				`	"How long until the test?"`
			`	Aliza checks a clock. "We've got most of the day. Want to help us set up, question?"`
			`	"Sure, I've got nothing better to do," you say.`

> 			`	What follows is a surprisingly mundane day of helping the crew set up various sensors and instruments at carefully marked distances from ground zero. As you're helping drill a borehole through the ice, through which yet another probe will be dropped, you see a sled in the distance drive by, an innocent-looking teardrop object strapped to its bed. It can't be any bigger than you yourself are, yet this must be the device that will be tested in only a few hours, raised on top of a slim steel tower that has already been erected.`

Reviewer 2: ```suggestion
			`	What follows is a surprisingly mundane day of helping the crew set up various sensors and instruments at carefully marked distances from ground zero. As you're helping drill a borehole through the ice, through which yet another probe will be dropped, you see a sled in the distance drive by, an innocent-looking teardrop-shaped object strapped to its bed. It can't be any bigger than you yourself are, yet this must be the device that will be tested in only a few hours, raised on top of a slim steel tower that has already been erected.`
```

			`	She looks suddenly embarrassed. "Worry. But you didn't hear that from me, of course. Information security is a lot more relaxed now that we're all mostly on the same side, but still... can't hurt to be safe."`

			label boom
			`	A few hours later, and a quick skim over Dokimi's surface in an Akoustiki Excursion Vehicle, you find yourself in a bunker dug into the ice a few dozen kilometers away. Up in the mountains, you have direct line of sight to the range, which sits just on the horizon.`
			`	You stand in silence, hunched over in a bunker not designed to accommodate one of your stature. A few Avgi shuffle awkwardly and try to make small talk, but it invariably trails away into the silence. Someone near the back coughs, an odd whistling sound, as you all stare at the room's sole window, tinted dark enough so as to be black. Finally, over the hum of electric lights, a nameless technician begins counting down together with the clock. "Twelve, eleven, ten, nine..."`
			`	You find yourself holding your breath as the countdown inches to zero. "Three, two, one...`
			choice
				`	"...Zero."`

> 			`	For a moment, silence. And then, light. Light. Nothing but light, a light that consumes your whole world even as the tinted glass shields your eyes from all but the barest glimpse of its edges. For a handful of seconds, a time that stretches into a lifetime, your whole world is light. Then it dims by a bare margin, merely blinding instead of consuming, and you can tell there is something outside the window other than infinite light, the border where the bunker ends and the landscape begins. And the landscape is illuminated, lit by a sun crafted by mortal hands. A sphere of brilliance rests in the distance, utterly still as it only just touches the planetoid's surface. As it dims with each passing moment, within the new sun you swear you can see something opaque and hard-edged, almost as if it were solid, a shape that burns with primordial fire.`

Reviewer 2: This is genuinely really beautiful writing.

			choice
				`	"I guess so."`
					goto scam
				`	"Whatever you're selling, I'm not interested."`

			`	You try to step around him and continue on your way, but he mirrors you, dancing backwards to remain in your face. "Sincerity! You are making a terrible mistake! I have something that could help with your energy, yes?"`
			choice
				`	"Alright, alright. What is it?"`
					goto scam
				`	"Not interested. Now move before I call the authorities."`
					goto nope

			label scam
			`	"Excitement! Wonderful! I have just the thing, certified by the Consonance Health Committee itself, surely!" he says animatedly, arms and wings waving as he dances in front of you. "But, reservation. Hmm. Only the most enterprising of captains could benefit from what I am about to bestow upon you. Are you certain you have what it takes to try Zapchannel Ray Energy?"`
			choice
				`	"Yes, I'm very experienced, seen the galaxy and all. Now show me what you're talking about."`
					goto glow
				`	"Alright, that's it. Get out of here before I call the authorities."`
					goto nope

			label glow

> 				`"Excitement, excellent! I can see you are clearly a very intelligent and discerning captain. Anticipation. This serum was invented by yours truly, capable of resolving all of your worldly woes! Only 3,000 credits!" He proudly holds out a vial of cloudy white liquid, glowing a bright, intense pale blue.`

Reviewer: ```suggestion
				`	"Excitement, excellent! I can see you are clearly a very intelligent and discerning captain. Anticipation. This serum was invented by yours truly, capable of resolving all of your worldly woes! Only 3,000 credits!" He proudly holds out a vial of cloudy white liquid, glowing a bright, intense pale blue.`
```
```suggestion
				`"Excitement, excellent! I can see you are clearly a very intelligent and discerning captain. Anticipation. This serum was invented by yours truly, capable of resolving all of your worldly woes! Only 3,000 credits!" He proudly holds out a vial of cloudy white liquid, glowing a bright, intense pale blue.`
```

			`	Eventually, Claret is ready to leave. "Anyway, I'll see you around. And, a word of advice. If you're ever on a Twilight Guard station, look out for officers or Guardsmen who are looking for a ride. We often have people trying to make an impromptu trip, and our logistics have been so overstretched lately they're willing to contract with outside captains, if they're trustworthy."`
			`	"I'll keep that in mind," you say.`
				decline

mission "Avgi: Atomics Testing 1"
	name "Atomic Testing"
	description "Take Aliza and her colleagues to the surface of Dokimi."
	landing
	passengers 6
	to offer
		not "avgi: lost in twilight"
	source "Ergastirio Station"
	destination "Dokimi Range"
	blocked "You need <capacity> to transport Aliza and her colleagues to Dokimi."
	on offer

> 		log "People" "Aliza" `Alizarin, or Aliza, is a scientist working on Ergastirio Station. She specializes in nuclear weapons design and testing, specifically in shockwave dynamics and accelerator testing. With a bit of an enthusiasm for nuclear weapons, she is eager to share what she knows.`

Reviewer 2: ```suggestion
		log "People" "Aliza" `Alizarin, or Aliza, is a scientist working on Ergastirio Station. She specializes in nuclear weapons design and testing, specifically in shockwave dynamics and accelerator testing. With a bit of a passion for nuclear weapons, she is eager to share what she knows.`
```

			choice
				`	"Uh, what is this?"`
				`	"Is this radioactive?"`
				`	"I'm not drinking that."`

			`	"Dismissal. Nonsense, this formula is perfectly safe! A serum for the health, proven to kill over three thousand forms of cancer, you should know." As if on cue, the glow pulses menacingly.`
			choice
				`	"Yeah, that doesn't look safe. I'm calling the authorities."`
					goto nope
				`	"Alright, I'll give it a shot."`

			branch superpowers
				random < 99
			`	You take the vial, warm to the touch, and uncork it. Nikkio gives you an expectent look, and gestures at you before you carefully tip the drink back. You feel a little tingly as the drink trickles down your throat, tasting vaguely of lemon and metal. This quickly intensifies into pins and needles in your chest, spreading as your body starts to go numb. As your vision darkens, punctuated by flashes of light in the corners of your eyes, your last thought is that perhaps you shouldn't have drunk a mysterious glowing liquid given to you by a seedy alien you just met in the spaceport.`
				die

			label superpowers

> 			`	You take the vial, warm to the touch, and uncork it before carefully tipping it back. You feel a little tingly as the drink trickles down your throat, tasting vaguely of lemon and metal. This quickly intensifies into pins and needles in center of your chest, spreading as you start to feel somehow revitalized. After a brief, sharp pain, a suspiciously dragonfly-shaped patch of skin in the center of your skin begins to glow a vibrant blue.`

Reviewer 2: ```suggestion
			`	You take the vial, warm to the touch, and uncork it before carefully tipping it back. You feel a little tingly as the drink trickles down your throat, tasting vaguely of lemon and metal. This quickly intensifies into pins and needles in the center of your chest, spreading as you start to feel somehow revitalized. After a brief, sharp pain, a suspiciously dragonfly-shaped patch of skin in the center of your skin begins to glow a vibrant blue.`
```

				`	"What about the Dissonance? I've heard of them."`
					goto dissonance

> 			`	"Contemplation. Perhaps a better question would be, 'what it was like before the hyperdrive?' For our society was not placed into turmoil first by the Aberrant, but rather by the discovery of a means to travel faster than light. Inside a handful of years an outlandish physics experiment went from theory to reality, and stars that had been decades or centuries away instead became reachable in first months, then weeks, then days as the hyperdrive continued to improve."`

Reviewer: ```suggestion
			`	"Contemplation. Perhaps a better question would be, 'what it was like before the hyperdrive?' For our society was not placed into turmoil first by the Aberrant, but rather by the discovery of a means to travel faster than light. Inside a handful of years an outlandish physics experiment went from theory to reality, and stars that had been decades or centuries away instead became reachable in first months, then weeks, then days as the hyperdrive continued to improve.`
```

Reviewer: It's a rule; if there are multiple paragraphs of dialog by the same person all except for the last paragraph should lack the ending quotation mark.

> 			`	"You must understand, emphasis, just how economically separated the various pockets of Avgi civilization were. Aviskir of course was dominant within the Anax system, but beyond interplanetary politics practically every system was independent in both a political and economic sense. There was no Consonance to organize the various powers, and everyone largely kept to themselves by sheer virtue of the distance and expense involved in interstellar travel."`

Reviewer: ```suggestion
			`	"You must understand, emphasis, just how economically separated the various pockets of Avgi civilization were. Aviskir of course was dominant within the Anax system, but beyond interplanetary politics practically every system was independent in both a political and economic sense. There was no Consonance to organize the various powers, and everyone largely kept to themselves by sheer virtue of the distance and expense involved in interstellar travel.`
```

			label superpowers
			`	You take the vial, warm to the touch, and uncork it before carefully tipping it back. You feel a little tingly as the drink trickles down your throat, tasting vaguely of lemon and metal. This quickly intensifies into pins and needles in center of your chest, spreading as you start to feel somehow revitalized. After a brief, sharp pain, a suspiciously dragonfly-shaped patch of skin in the center of your skin begins to glow a vibrant blue.`
			`	"Surprise. You surely are, ah, a worthy captain!" says Nikkio, shock evident on his face. He begins to sidle away, glancing in all directions as though he isn't sure what's going on. "Hurry. I must be on my way, goodbye for now, captain!"`

> 			accept

Reviewer: Tab.

					goto actually
						or

> 							"has: FW Syndicate Capture 1B: offered

Reviewer: ```suggestion
							"has: FW Syndicate Capture 1B: offered"
```

				`	"Terrifying."`

			`	"Contemplation. Indeed it is." You turn and see Aliza watching the fireball with you, along with a few others. The rest are dutifully staring at instrument readouts and sensor logs, collecting whatever esoteric knowledge they sought to gain from this test.`
			choice
				`	"So, how did it go?"`

> 				`	"What were you trying to learn form this?"`

Reviewer: ```suggestion
				`	"What were you trying to learn from this?"`
```
